tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing.md

```diff
@@ -158,8 +157,18 @@ cargo test --package custom-pallet
 
 After running the test suite, you should see the following output in your terminal:
 
---8<-- 'code/tutorials/polkadot-sdk/parachains/build-custom-pallet/pallet-unit-testing/output.html'
+--8<-- 'code/tutorials/polkadot-sdk/parachains/zero-to-hero/pallet-unit-testing/output.html'
 
 ## Where to Go Next
 
-Expand your Polkadot SDK development skills by exploring the [`Pallet Benchmarking`](/tutorials/polkadot-sdk/parachains/build-custom-pallet/pallet-benchmarking){target=\_blank} tutorial.
+<div class="grid cards" markdown>
+
+-   <span class="badge tutorial">Tutorial</span> __Add Pallets to the Runtime__
+
+    ---
+
+    Add pallets to your runtime for custom functionality. Learn to configure and integrate pallets in Polkadot SDK-based blockchains.
+
+    [:octicons-arrow-right-24: Add Pallets to the Runtime](/tutorials/polkadot-sdk/parachains/zero-to-hero/add-pallets-to-runtime/)
```

Comment:
Don't repeat the title as the CTA. We can use something generic here like "Get started"